Packs/ServiceNow/Integrations/ServiceNowv2/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+  - Clarified the documentation for usage of the ***impact*** and the ***urgency*** arguments in the commands:
+    - ***servicenow-update-ticket***
+    - ***servicenow-create-ticket***
```

Comment:
```suggestion
  - Improved documentation regarding the usage of the ***impact*** and the ***urgency*** arguments for the commands:
    - ***servicenow-update-ticket***
    - ***servicenow-create-ticket***
```

Reply:
I'm inclined to agree with @liorblob. The most important message to convey is that the end-user can select from the prepopulated values, or override it using some sort of template. 

I would suggest the following:

> 'Ticket urgency. You may select from the predefined options, or define a custom field by using the following syntax. "Number - Label". (For example, "4 - Medium")